docs/UserGuide.md

```diff
@@ -264,20 +194,100 @@ __________________________________________
 ```
 
 
+### List all items in Fridget: `list`
+List all items stored in Fridget.
+
+Format: list -<OPTIONAL_SORT_TYPE> or list <OPTIONAL_SORT_TYPE>
+
+* `list` Lists in Alphabetical order of the Item Name.
+* `list -e` Lists items in ascending order of Expiry Dates.
+* `list -r` Lists items in descending order the item entered the ledger.
+
+Example of usage:
+
+```markdown
+userInput: list
+__________________________________________
+List sorted by item name:
+    1. fish Cake | Qty: 2 | 12 Dec 2020
+    2. frozen duck | Qty: 1 | 10 Oct 2022
+    3. Yoghurt cake | Qty: 1 | 15 Oct 2021
+__________________________________________
+```
+```
+userInput: list -e
+__________________________________________
+List sorted by expiry date:
+    1. fish Cake | Qty: 2 | 12 Dec 2020
+    2. Yoghurt cake | Qty: 1 | 15 Oct 2021
+    3. frozen duck | Qty: 1 | 10 Oct 2022
+__________________________________________
+```
+```
+userInput: list -r
+__________________________________________
+List sorted by earliest added:
+    1. fish Cake | Qty: 2 | 12 Dec 2020
+    2. Yoghurt cake | Qty: 1 | 15 Oct 2021
+    3. frozen duck | Qty: 1 | 10 Oct 2022
+__________________________________________
+```
+
+### Find items stored in Fridget: `find`
+Find all items associated with the keyword.
+
+Format: find <KEYWORD>
+
+* The `KEYWORD` should be something associated with the ingredient.
```

Comment:
Can we reword this to:
* The `KEYWORD` should be the same as the name of the item you are looking for. 
* The `KEYWORD` can also be a part of the name of the item.